VEHICLE/N_0x8aa9180de2fedd45.md

```diff
@@ -1,15 +1,19 @@
 ---
 ns: VEHICLE
 ---
-## _0x8AA9180DE2FEDD45
+## EnableAircraftObstacleAvoidance
```

Comment:
Hi!
If you're renaming the header, you should also rename the function method (at line 8) to match this.

Reply:
Also this isn't the proper way to rename this, the name should be UPPER_SNAKE_CASE